app/helpers/workflow_helper.rb

```diff
@@ -4,7 +4,7 @@ def show_workflow_grid?
   end
 
   def render_workflow_grid
-    workflow_data = facet_tree('wf_wps')['wf_wps_ssim']
+    workflow_data = params[:all] ? facet_tree : facet_tree('wf_wps')['wf_wps_ssim']
```

Comment:
The `:all` parameter intends to reset the facets to include all items (akin to "Start Over"). Is there a better way to reset the facets in Blacklight?